### ISSUE TITLE
Add -s flag for gofmt parity

### ISSUE THREAD

**Issue by johanbrandhorst** [sqs/goreturns#45]
(gofmt)[https://golang.org/cmd/gofmt/] offers a `-s` parameter for simplifying code, which is quite nice. I think goreturns should add this parameter as well since it promises to support a superset of gofmt's features, and I'd like _both_ simplified code and auto-returns.

**Comment by sqs** [comment]
Yes, this would be great. Overall I would like to overhaul goreturns soon and make it shell out to goimports/gofmt and (probably) pass along flags to future-proof it.

A PR for this would be gladly accepted to get this in sooner, though!

**Comment by MaerF0x0** [comment]
is this still the planned path? I can see if I can handle `-s`
